### ISSUE TITLE
Inbox - Group tab should list group display name if available

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/HPI#1106]
I have a group on Edge2 with the following setup:

- group ID: GROUP_escalated_uw_reviewers
- group display name: Escalated Underwriter Reviewers

When displaying the group tab on the inbox, we should use the pretty name if it's set and fall back to the group ID only if the display name is blank:

![image](https://cloud.githubusercontent.com/assets/1377228/15611963/6c1a14d0-23f2-11e6-80e0-9eee01733fc3.png)

**Comment by cmlewis** [comment]
Because this is straight Activiti API and this is the actual attribute value on the wf, we'll have to come up with repo impls to get display names.

**Comment by gsteimer** [comment]
Yeah - I'm guessing that you would just use `IUser#searchAuthorities` to search for the group and then pull of the group display name from the `OCGroup` object.
